ext/rugged/rugged_tree.c

```diff
@@ -44,7 +44,7 @@ static VALUE rb_git_treeentry_fromC(const git_tree_entry *entry)
 
 	rb_entry = rb_hash_new();
 
-	rb_hash_aset(rb_entry, CSTR2SYM("name"), rb_str_new_utf8(git_tree_entry_name(entry)));
+	rb_hash_aset(rb_entry, CSTR2SYM("name"), rb_str_new2(git_tree_entry_name(entry)));
```

Comment:
This is tangential to this PR, but I noticed that `git_tree_entry` has a length field for the name, but no accessor function.  Should we add an accessor function and use the length to create the string here?